src/api/models/base_model_.py

```diff
@@ -0,0 +1,80 @@
+"""API Models Logic."""
+
+# Standard Python Libraries
+import pprint
+import typing
+
+# Third-Party Libraries
+import six  # type: ignore
+
+# cisagov Libraries
+from api import util
+
+CustomType = typing.TypeVar("CustomType")
+
+
+class Model:
+    """Mode Class."""
```

Comment:
Typo/caps, but you really should come up with a more informative docstring:
```suggestion
    """Model class."""
```